src/lib/web3-utils.js

```diff
@@ -25,9 +25,11 @@ export function getAppPresentation(apps, appAddress) {
   const { contentUri, manifest } = apps.find(
     ({ address }) => address === appAddress
   )
-
-  const iconPath = manifest.icons[0].src
-  const iconSrc = getIpfsUrlFromUri(contentUri) + iconPath
+  let iconSrc = ''
+  if (manifest.icons && manifest.icons[0]) {
+    const iconPath = manifest.icons[0].src
+    iconSrc = getIpfsUrlFromUri(contentUri) + iconPath
+  }
```

Comment:
Is this change supposed to be here? I can't see how it applies.

Reply:
Yes, sorry i put it here cause i change the org to the one that we will be  using and some of them doesnt have manifest.icons so it was breaking